enhancements/machine-api/aws_efa_networking.md

```diff
@@ -0,0 +1,253 @@
+---
+title: aws-efa-networking
+authors:
+  - "@JoelSpeed"
+reviewers:
+  - "@elmiko"
+  - "@deads2k"
+  - TBD
+approvers:
+  - TBD
+api-approvers:
+  - "@deads2k"
+creation-date: 2021-12-15
+last-updated: 2021-12-15
+tracking-link:
+  - https://issues.redhat.com/browse/OCPCLOUD-1353
+---
+
+# AWS EFA Networking
+
+## Summary
+
+AWS Elastic-Fabric-Adapter (EFA) is a feature within EC2 that allows for high throughput network adapters to be
+attached to certain instance types. With the presence of these network adapters and some additional drivers,
+workloads can take advantage of greatly improved network performance between nodes within the same AWS availability
+zone.
+
+## Motivation
+
+We have customers already using this feature of EC2 within OpenShift UPI clusters. They have taken on the responsibility
+of making sure the prerequisites are fulfilled and have manually created instances with the attached adapters.
+However, there is a desire to enable this feature to be used in conjunction with existing autoscaling features within
+OpenShift. To enable this, we must provide support for the feature within the Machine API, enabling IPI clusters to also
```

Comment:
Rather than mention IPI here just focus on autoscaling since both UPI and IPI commonly make use of MAPI in AWS.